src/integrationTest/java/uk/gov/hmcts/darts/ws/AddAudioWebServiceTest.java

```diff
@@ -26,4 +39,15 @@ void addAudio(DartsGatewayClient client) throws Exception {
         response.assertIdenticalResponse(client.convertData(expectedResponseStr, AddAudioResponse.class).getValue());
     }
 
+    @ParameterizedTest
+    @ArgumentsSource(DartsClientProvider.class)
+    void addAudioHandleError(DartsGatewayClient client) throws Exception {
+        String soapRequestStr = TestUtils.getContentsFromFile(
+            "payloads/addAudio/register/invalidSoapRequest.xml");
+
+        Assertions.assertThatExceptionOfType(SoapFaultClientException.class).isThrownBy(() -> {
```

Comment:
Once the above mapping is implemented we can then test for the associated SOAP errors based on each of thea add audio darts api problem error codes